src/ray/util/event.cc

```diff
@@ -253,28 +273,39 @@ void RayEvent::SendMessage(const std::string &message) {
                                        ? RayEventContext::Instance()
                                        : RayEventContext::GlobalInstance();
 
-  rpc::Event event;
-
-  std::string event_id_buffer = std::string(18, ' ');
-  FillRandom(&event_id_buffer);
-  event.set_event_id(StringToHex(event_id_buffer));
-
-  event.set_source_type(context.GetSourceType());
-  event.set_source_hostname(context.GetSourceHostname());
-  event.set_source_pid(context.GetSourcePid());
-
-  event.set_severity(severity_);
-  event.set_label(label_);
-  event.set_message(message);
-  event.set_timestamp(current_sys_time_us());
+  static const int event_id_size = 18;
```

Comment:
nits: prefer kEventIDSize for constant
https://google.github.io/styleguide/cppguide.html#Constant_Names